src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  @include smooth;
+
+  border: solid 1px #f3f3f3;
+  width: $card-width;
```

Comment:
Make sure that the `$card-width` variable is defined in your SCSS files. If it's not, you need to define it to avoid compilation errors.